src/OpenTelemetry/Logs/OpenTelemetryLoggerProvider.cs

```diff
@@ -188,6 +194,8 @@ public bool ForceFlush(int timeoutMilliseconds = Timeout.Infinite)
         /// <returns>The supplied <see cref="OpenTelemetryLoggerOptions"/> for chaining.</returns>
         public OpenTelemetryLoggerProvider AddProcessor(BaseProcessor<LogRecord> processor)
         {
+            OpenTelemetrySdkEventSource.Log.OpenTelemetryLoggerProviderEvent("Started adding processor.");
```

Comment:
What I would really like to see here is more details about the specific configuration of the processor. For example, simple or batch processor. Type of the exporter (if there is one). Could probably do this through `ToString` operations, I think that is what is done in the MeterProviderSdk logging. Also, could we add to the logging whether or not the shared or static pool is selected?

I think this will be important as we move away from config being done strictly by code. For example through dependencies or in the future this might be done through an external config file (@alanwest is looking at that).

Reply:
I hope it is good now.